stablehlo/dialect/VhloTypes.td

```diff
@@ -86,6 +86,12 @@ def VHLO_FloatF8E4M3FNV1 : VHLO_TypeDef<"FloatF8E4M3FNV1", "f8E4M3FN_v1", "0.9.0
 // Corresponds to the 'f8E5M2' FloatType from the StableHLO spec.
 def VHLO_FloatF8E5M2V1 : VHLO_TypeDef<"FloatF8E5M2V1", "f8E5M2_v1", "0.9.0", "current">;
 
+// Corresponds to the 'f8E4M3FNUZ' FloatType from the StableHLO spec.
+def VHLO_FloatF8E4M3FNUZV1 : VHLO_TypeDef<"FloatF8E4M3FNUZV1", "f8E4M3FNUZ_v1", "0.9.0", "current">;
+
+// Corresponds to the 'f8E5M2FNUZ' FloatType from the StableHLO spec.
+def VHLO_FloatF8E5M2FNUZV1 : VHLO_TypeDef<"FloatF8E5M2FNUZV1", "f8E5M2FNUZ_v1", "0.9.0", "current">;
+
```

Comment:
@GleasonK Looks like the time has come to figure out the versioning scheme moving forward (similarly to what we discussed in #1319). What would be your recommendation?

Reply:
This predicts exactly one of my questions I had for you guys 👍

Reply:
Just seeing this. Added a separate comment with a few steps

Reply:
@GleasonK I assume point 2 also means updating places that refer to `0.9.0` in the present tense?
Examples of what I mean:
https://github.com/openxla/stablehlo/blob/7f6976e15595d3a37ad177a58b4a3c2024d9ae9c/docs/bytecode.md?plain=1#L24
https://github.com/openxla/stablehlo/blob/7f6976e15595d3a37ad177a58b4a3c2024d9ae9c/docs/compatibility.md?plain=1#L10

Reply:
I think I've covered these items. Let me know if there's anything I've missed wrt. versioning.